gulpfile.js

```diff
@@ -78,10 +79,13 @@ const getBrowserifyInstance = function() {
 
 // receives a browserify instance and bundles it
 const bundleBrowserify = function(b) {
-  b
-    .transform(babelify)
```

Comment:
Really glad you found this optimization but it needs to be added to the `browserify` task in the same way that you did for `watchify`. Right now `browserify` breaks when it comes across JSX because it doesn't use the transform anymore.